shlr/capstone.sh

```diff
@@ -1,58 +1,84 @@
-#!/bin/sh
+#! /bin/sh
```

Comment:
why space is here?

Reply:
Space is [allow](https://unix.stackexchange.com/questions/276751/is-space-allowed-between-and-bin-bash-in-shebang) after `#!`. But if you prefer not having space after `#!`, I will change it.